qiskit_ibm_runtime/utils/noise_learner_result.py

```diff
@@ -231,7 +231,7 @@ def draw_map(
         background_color: str = "white",
         radius: float = 0.25,
         width: int = 800,
-    ) -> go.Figure:
+    ) -> "PlotlyFigure":
```

Comment:
```suggestion
    ) -> PlotlyFigure:
```
This should work, try it out